tck/src/main/java/org/eclipse/microprofile/openapi/apps/airlines/resources/UserResource.java

```diff
@@ -279,7 +276,7 @@ public Response getUserByName(
                                parameters = @LinkParameter(name = "userId", expression = "$request.path.id"),
                                extensions = @Extension(name = "x-link", value = "test-link")),
                          @Link(name = "Review", description = "The reviews provided by user",
-                               operationRef = "/db/reviews/{userName}",
+                               operationRef = "#/paths/~1reviews~1users~1{user}/get",
                                parameters = @LinkParameter(name = "path.userName",
                                                            expression = "$response.body#userName"),
```

Comment:
```suggestion
                               operationRef = "#/paths/~1reviews~1users~1{user}/get",
                               parameters = @LinkParameter(name = "path.user",
                                                           expression = "$response.body#userName"),
```
Since the path parameter is named `user`.